.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
no need to use lint-staged, you can use `npm run lint` from here

Reply:
Also, update the lint script in package.json to ` "eslint --report-unused-disable-directives .",`